dataprofiler/tests/profilers/test_column_profile_compilers.py

```diff
@@ -785,34 +785,38 @@ def test_json_decode_after_update(self, mock_instance):
         # assert before update
         assert deserialized.report().get("data_label", None) == "a|b"
         assert (
-            sum(
-                [
-                    v
-                    for k, v in deserialized.report()
-                    .get("statistics", None)
-                    .get("avg_predictions", None)
-                    .items()
-                ]
-            )
-            == 1.0
+            deserialized.report()
+            .get("statistics", None)
+            .get("data_label_representation", None)
+            .get("a", None)
+            == 0.5
+        )
+        assert (
+            deserialized.report()
+            .get("statistics", None)
+            .get("data_label_representation", None)
+            .get("b", None)
+            == 0.5
         )
```

Comment:
can we instead assert the list and do one assert.
```python
assert (deserialized.report()
    .get("statistics", None)
    .get("data_label_representation", None)
    == {"a": 0.5, "b": 0.5}
)
```

Reply:
pre-commit would fix

Reply:
Note that `.get("statistics", {})` is critical as otherwise if `statistics` doesn't exist this will cause an error.

Reply:
updated